R/PackageSummaryReporter.R

```diff
@@ -14,7 +14,11 @@ PackageSummaryReporter <- R6::R6Class(
             system.file(file.path("package_report","package_summary_reporter.Rmd"),package = "pkgnet")
         },
         get_description = function(){
-            return(private$packageDescription)
+            obj <- private$packageDescription
+            dt <- data.table::data.table(Field = names(obj)
```

Comment:
We should not be naming objects `dt`, since that conflicts with the `dt` function available by default in all R sections. Can you please give this a more descriptive name?